cfdm/read_write/netcdf/netcdfwrite.py

```diff
@@ -1627,9 +1628,11 @@ def _write_node_count(self, coord, bounds, coord_ncdimensions, encodings):
             self.implementation.get_data(bounds)
         )
         if self.implementation.get_data_ndim(bounds) == 2:  # DCH
-            array = numpy.ma.count(array, axis=1)  # DCH
+            array = np.ma.count(array, axis=1)  # DCH
         else:  # DCH
-            array = numpy.ma.count(array, axis=2).sum(axis=1)  # DCH
+            array = np.ma.count(array, axis=2).sum(axis=1)  # DCH
```

Comment:
What are the `DCH` tags here for? Notably are they TODOs for now or for later, just to check in case you were going to amend or add something to the PR that might have been missed?

Reply:
I wish I new!  Probably that I'm not sure that these lines are correct. Let's leave them there and I'll make a note to review.

Reply:
Ah wait - ignore me - I thought you had added those, wasn't reading the `diff` carefully enough!